### PR TITLE
Move cap1188 to drivers folder.

### PR DESCRIPTION
**Pull request by caternuson**
Accidentally put the CAP1188 driver one folder up. This PR moves it to the correct location and updates `.gitmodules`.

### CODE REVIEW

**Comment by caternuson** [comment]
Not sure how that happened. Manually cleaned up. Hopefully this is OK now.

File: .gitmodules [hunk 1]
```diff
@@ -301,8 +301,8 @@
 [submodule "libraries/drivers/lidarlite"]
 	path = libraries/drivers/lidarlite
 	url = https://github.com/adafruit/Adafruit_CircuitPython_LIDARLite.git
-[submodule "libraries/cap1188"]
-	path = libraries/cap1188
+[submodule "libraries/drivers/cap1188"]
+	path = libraries/drivers/cap1188
 	url = https://github.com/adafruit/Adafruit_CircuitPython_Bundle.git
```
[the remark this answers is not in the document]
Comment (caternuson) [reply .gitmodules]: it totally is

**Comment by caternuson** [comment]
This PR is just all kinds of messed up. Sending it to the dungeon.

**Comment by kattni** [comment]
Closed in favor of #99

**Pull request closed without merging**